buildtest/buildsystem/base.py

```diff
@@ -368,14 +369,14 @@ class data structures that are used by both run and dry_run
         self.history = {}
         self.history["TESTS"] = []
 
-        # Metadata includes known sections in a config recipe (pre/post_run, env)
-        # These should all be validated for type, format, by the schema validator
+        # Metadata includes known sections in a Buildspec (pre/post_run, env)
+        # These should all be validated for type, format, by the Buildspec schema
         self.metadata = {}
         for known_section in known_sections:
             if known_section in self.recipe:
                 self.metadata[known_section] = self.recipe.get(known_section)
 
-        # Every build recipe has a shell (defaults to BUILDTEST_SHELL_DEFAULT
+        # Every Buildspec recipe has a shell (defaults to BUILDTEST_SHELL_DEFAULT)
```

Comment:
Is this true?

Reply:
hmm i guess according to schema https://github.com/HPC-buildtest/schemas/blob/master/script/script-v0.0.1.schema.json#L44 it defaults to ``bash`` there is no enum on ``shell`` so user can specify anything. 

I think the comment can be a bit misleading, every Buildspec recipe doesn't need to declare a ``shell``. It defaults to ``bash`` which is ``BUILDTEST_SHELL_DEFAULT``.